lib/hashauth.js

```diff
@@ -170,7 +172,7 @@ hashauth.init = function init(client, $) {
     var html =
       '<div id="requestauthenticationdialog" style="display:none" title="'+translate('Device authentication')+'">'+
         '<label for="apisecret">'+translate('Your API secret')+': </label>'+
-        '<input type="password" id="apisecret" size="20" />'+
+        '<input type="password" id="apisecret" size="20" style="width: 100%;"/>'+
```

Comment:
make a secret input element to be a full width, because the secret could be longer than 20 chars.